packages/snap-platforms/common/README.md

```diff
@@ -2,11 +2,101 @@
 This platform library gives you helper functions to use for all integrations regardless of the platform. 
 
 
-## Usage 
+## Snap Usage 
 To use the platform library, simply import what you wish to use from `@searchspring/snap-platforms/common`.
 
 ```jsx
-import { pluginLogger } from '@searchspring/snap-platforms/common';
+import { scrollToTop } from '@searchspring/snap-platforms/common';
+const scrollToTopConfig = {
+    enabled: true
+}
+controller.plugin(scrollToTop, scrollToTopConfig);
+```
 
-controller.plugin(pluginLogger);
+## SnapTemplates Usage
+To use a plugin in [SnapTemplates](https://github.com/searchspring/snap/blob/main/docs/TEMPLATES_ABOUT.md), it can be defined and configured in the `config.plugins` section.
+
+```jsx
+plugins: {
+	common: {
+		scrollToTop: {
+			enabled: true
```

Comment:
Maybe since this is always attached and enabled we should change the example here to set a different scroll to element?